### PR TITLE
feat: did support

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx [hunk 1]
```diff
@@ -1,30 +1,42 @@
...
-export const getIdentityVerificationText = (verificationStatus: VerificationFragment[]): string => {
+export const getIdentityVerificationText = (
+  verificationStatus: VerificationFragment[],
+  document: WrappedDocument<v2.OpenAttestationDocument>
+): string => {
+  const data = getData(document);
   const registryFragment = verificationStatus.find((status) => status.name === "OpencertsRegistryVerifier");
-  const dnsFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsTxt");
+  const dnsFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsTxtIdentityProof");
+  const dnsDidFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsDidIdentityProof");
```
Comment (simboonlong) [inline src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx:17]: not sure if u want to be more explicit here on variable naming
```suggestion
  const dnsTxtFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsTxtIdentityProof");
  const dnsDidFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsDidIdentityProof");
```

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx [hunk 2]
```diff
@@ -1,30 +1,42 @@
...
 
   // using concat to handle arrays and single element
   const registryFragmentData: OpencertsRegistryVerificationFragmentData[] = [].concat(registryFragment?.data || []);
   const dnsFragmentData: (DnsFragment | undefined)[] = [].concat(dnsFragment?.data || []);
+  const dnsDidFragmentData: (DnsFragment | undefined)[] = [].concat(dnsDidFragment?.data || []);
```
Comment (simboonlong) [inline src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx:22]: yeah might have quite a few places to change for the naming... im fine if it's not changed then

**Review by simboonlong — APPROVED** [review]

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx (hunk 1 above)
Comment (Nebulis) [reply src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx]: done

File: src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx (hunk 2 above)
Comment (Nebulis) [reply src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx]: done

**Pull request merged**